rpc_jobs/rpc_aio.yml

```diff
@@ -225,8 +226,10 @@
             }} catch (e) {{
               print(e)
               throw e
+              currentBuild.result = 'FAILURE'
```

Comment:
Line 228 throws an exception, if line 229 still executes, when does the the flow switch to the next applicable catch block?